src/chunkserver/io_buffers.cc

```diff
@@ -436,3 +440,32 @@ void InputBuffer::setFinished() { state_.store(WriteState::Finished); }
 bool InputBuffer::isHeaderSizeValid() const {
 	return headerBuffer_.totalBytesPutInBuffer() == writeInfo_.size() * headerSize_;
 }
+
+void releaseOldIoBuffers(uint32_t expirationTime_ms) {
+	auto initialTotalOutputBufferBlocks = gCurrentTotalOutputBufferBlocks.load();
+	auto initialTotalInputBufferBlocks = gCurrentTotalInputBufferBlocks.load();
+	auto initialTotalReplicatorBufferBlocks = gCurrentTotalReplicatorBufferBlocks.load();
+
+	getReadOutputBufferPool().releaseOldBuffers(expirationTime_ms);
+	getWriteInputBufferPool().releaseOldBuffers(expirationTime_ms);
+	getReplicateBuffersPool().releaseOldBuffers(expirationTime_ms);
+
+	// Calculate the number of released blocks. Please note that the number of released
+	// could be negative because some buffers could be added to the pools in the meantime.
+	int releasedOutputBuffers =
+	    initialTotalOutputBufferBlocks - gCurrentTotalOutputBufferBlocks.load();
+	int releasedInputBuffers =
+	    initialTotalInputBufferBlocks - gCurrentTotalInputBufferBlocks.load();
+	int releasedReplicatorBuffers =
+	    initialTotalReplicatorBufferBlocks - gCurrentTotalReplicatorBufferBlocks.load();
+	// Log the number of released buffers.
+	safs::log_debug(
+	    "(releaseOldIoBuffers) Released buffer blocks per operation: read {}, write {}, replicate {}",
```

Comment:
Consider using `__func__` macro to avoid typing manually the name of the function.
```suggestion
	    "({}) Released buffer blocks per operation: read {}, write {}, replicate {}",
	    __func__, gCurrentTotalOutputBufferBlocks.load(), gCurrentTotalInputBufferBlocks.load(),
```

Reply:
Applied here and in some other functions from the same PR.